desktop/api/src/main/java/org/datacleaner/panels/MutableInputColumnListPanel.java

```diff
@@ -51,7 +52,18 @@
     private final AnalysisJobBuilder _analysisJobBuilder;
     private final OutputColumnVisibilityButton _visibilityButton;
 
-    public MutableInputColumnListPanel(AnalysisJobBuilder analysisJobBuilder, MutableInputColumn<?> inputColumn) {
+    public JComponent panelOwner;
```

Comment:
Looks to me like this field is _always_ null. It is not set via the constructor.